torch/_meta_registrations.py

```diff
@@ -6049,8 +6049,10 @@ def meta_bucketize(self, boundaries, *, out_int32=False, right=False):
     ).contiguous()
 
 
-@register_meta(aten._upsample_bilinear2d_aa.default)
-def meta_upsample_bilinear2d_aa(
+@register_meta(
+    [aten._upsample_bilinear2d_aa.default, aten._upsample_bicubic2d_aa.default]
+)
+def meta_upsample_bimode2d_aa(
```

Comment:
Are we able to unskip interpolate in the meta tests now?
https://github.com/pytorch/pytorch/blob/5c700f60a5ee588d32975fc0acd80dfc06e14425/test/test_meta.py#L736